configure.kafkacat

```diff
@@ -43,8 +43,22 @@ struct rd_kafka_metadata foo;"
     then
         mkl_allvar_set "json" ENABLE_JSON y
     fi
+
+
+
+    mkl_meta_set "libserdes" "deb" "libserdes-dev"
+    # Check for Avro serdes library (serdes)
+    if [[ $WITH_AVRO == y ]] && \
+        mkl_lib_check --static=-lserdes "serdes" HAVE_SERDES disable CC "-lserdes -lavro" \
+        "#include  <stdlib.h>
+        #include  <libserdes/serdes-avro.h>
```

Comment:
It looks like on Mac you need to include 
#include <sys/types.h> 
as well or the test compile will fail.